docs/configuration/localization.mdx

```diff
@@ -200,7 +200,7 @@ locale as well as `'null'`, `'false'`, `false`, and `'none'`.
 const posts = await payload.find({
   collection: 'posts',
   locale: 'es',
-  fallbackLocale: false,
+  fallbackLocale: 'none',
```

Comment:
Passing a string of `'none'` seems gross to me. Can't we just make the type `false` and sanitize it to that from whatever we want for the search params? 'false', 'none', 'null', should could all be accepted to convert to `false`.